cmd/launch.go

```diff
@@ -260,7 +276,21 @@ func ValidateLaunchFlags(flags *config.SimpleInfo) bool {
 		fmt.Println("Error: You can't define the version without launch template")
 		return false
 	}
-
+	if flags.UserDataFilePath != "" {
+		_, err := os.Stat(flags.UserDataFilePath)
+		if err != nil {
+			fmt.Println("Error: Boot script file path invalid or does not exist")
```

Comment:
Consider using a single naming convention for user data  here and in other places - `userdata` `boot script`

Reply:
Yeah.. you're definitely right that the inconsistency can be confusing/weird. I was thinking of exposing user data as _bootScript_ to users, but just keep treating it as user data (since that's what it really is) within the code.

Any thoughts? Should we just go with calling it user data?

Reply:
sounds good - `userDataFile` `userDataFilePath` `userData` :)

Reply:
hmm.. i think i'm just going to with `BootScriptFilePath` and refer to it accordingly throughout code only referring to it as *UserData* when creating the RunInstance req.

My thinking here is:
* We assume this will always be a bash script (required for auto-termination)
* Slightly more clear to users less familiar with AWS
* I can use `-b` shorthand and have it make sense :) 

Lmk if you have any thoughts; otherwise, i'll move forward with it in next revision

Reply:
that makes sense 👍